HW6/kondntaliia/HW6_1.py

```diff
@@ -0,0 +1,9 @@
+def return_largest_number(number1, number2):
+    """
+    This function returns 
+    the largest number of two numbers.
+    """
+
+    return int(max([number1, number2]))
```

Comment:
perhaps - max([int(number1), int(number2)])

Reply:
Yes, that's better! Thank you!!!